core/dynamo.c

```diff
@@ -888,6 +888,7 @@ standalone_init(void)
 # ifdef DEBUG
     /* FIXME: share code w/ main init routine? */
     nonshared_stats.logmask = LOG_ALL;
+    /* Why do we need this to options_init()?  We did so above. */
```

Comment:
It does not look like `options_init` directly depends on anything set by the functions above, but maybe one of the functions it calls does. If this call is really needed, it might be problematic, as it replaces all options set previously by the default options. @derekbruening  do you know if it would be safe to drop this call?

Reply:
Out sick, will try to take a look tomorrow

Reply:
The -stats_logmask option sets stats->logmask and stats == nonshared_stats so there is a connection.  The default logmask has to be set before calling options_init().  I believe this was the original call, and the one above was added later.  I think the logmask setting on line 890 needs to be moved up above the options_init() on line 869, and then the options_init() here can be removed.

Reply:
Done.